projects/js-packages/publicize-components/src/social-store/types.ts

```diff
@@ -84,6 +84,7 @@ export interface KeyringResult extends KeyringAdditionalUser {
 	label: string;
 	service: string;
 	status: ConnectionStatus;
+	show_linkedin_warning: boolean;
```

Comment:
This can be optional

```suggestion
	show_linkedin_warning?: boolean;
```